dapp/src/store/mobx/state/price.js

```diff
@@ -0,0 +1,27 @@
+import { action, observable, flow, makeObservable } from 'mobx'
+import request from 'superagent'
+
+const tokenAddress = '0x970b9bb2c0444f5e81e9d0efb84c8ccdcdcaf84d'
```

Comment:
Cool, new mobx states!
Can you put the address into config?
We currently have a `FuseToken` there that is the CLN ><